models/issue.go

```diff
@@ -1062,15 +1062,15 @@ func UpdateIssueMentions(issueID int64, mentions []string) error {
 // IssueStats represents issue statistic information.
 type IssueStats struct {
 	OpenCount, ClosedCount int64
-	AllCount               int64
+	YourRepositoriesCount  int64
```

Comment:
It wasn't clear to me that you were asking that. I renamed AllCount to YourRepositoriesCount because AllCount sounds like it is the count of **all** issues open _in which you are participating_, **not** the count of open issues _in your repositories_ (which is what is actually displayed).

Before this Pull Request, the name "All" makes sense, since the "Assigned" and "Created" views show filtered versions of "All". After this Pull Request "Assigned" and "Created" views show different information compared to the "Your repositories" view (namely that it can also show issues from repositories not owned by you).